spec/item_spec.rb

```diff
@@ -0,0 +1,20 @@
+require './lib/item'
+
+RSpec.describe Item do
+  before(:each) do
+    @item = Item.new
+    @item1 = Item.new({name: 'Peach', price: "$0.75"})
```

Comment:
I love that you did test the default functionality; to be most clear, I would have added an expect line to my `exists` test to verify that `@item1` is also an Item.